modules/ROOT/pages/administration/databases.adoc

```diff
@@ -903,6 +925,76 @@ ALTER DATABASE nonExisting IF EXISTS SET TOPOLOGY 1 PRIMARY 0 SECONDARY
 [role="statsonlyqueryresult"]
 0 rows
 
+[role=enterprise-edition]
+[[administration-databases-alter-database-options]]
+=== Options
+
+The `ALTER DATABASE` command can be used to set or remove specific options.
+
+[NOTE]
+====
+There are no available `OPTIONS` values for composite databases.
+====
+
+
+[options="header"]
+|===
+
+| Key | Value | Description
+
+| `txLogEnrichment`
+| `FULL` \| `DIFF` \| `NONE`
+|
+Defines level of enrichment applied to transaction logs for Change Data Capture (CDC) purposes.
+
+|===
+
+.+ALTER DATABASE SET OPTION+
+======
+
+.Query
+[source, cypher, test-skip]
```

Comment:
I guess they have test-skip due to feature flag being off by default? should probably have a comment about why they are skipped anyway (if we can't run them/until we update so they can be run)